### ISSUE TITLE
Breaks external urls

### ISSUE THREAD

**Issue by rasteiner** [schnti/kirby3-cachebuster#5]
Hello, nice plugin! 

I sometimes use the `css` and `js` helpers to load external urls; like from public CDNs. 
The plugin doesn't distinguish between local and remote urls, therefore it tries to add modification timestamps also to external urls (which fails, and also if it would succeed, the url would then probably return a 404).

**Quick and dirty fix:**

Ignore the urls that specify a "host" part.

```diff

Kirby::plugin('schnti/cachebuster', [
	'options'    => [
		'active' => true
	],
	'components' => [
		'css' => function ($kirby, $url) {
-			if ($kirby->option('schnti.cachebuster.active')) {
+			if (!isset(parse_url($url)['host']) &&  $kirby->option('schnti.cachebuster.active')) {

				$file = $kirby->roots()->index() . DS . $url;
				return dirname($url) . '/' . F::name($url) . '.' . F::modified($file) . '.css';

			} else {
				return $url;
			}
		},
		'js'  => function ($kirby, $url) {
-			if ($kirby->option('schnti.cachebuster.active')) {
+			if (!isset(parse_url($url)['host']) && $kirby->option('schnti.cachebuster.active')) {

				$file = $kirby->roots()->index() . DS . $url;
				return dirname($url) . '/' . F::name($url) . '.' . F::modified($file) . '.js';

			} else {
				return $url;
			}
		}
	]
]);
```

**Comment by apps4research** [comment]
Thanks for this however, it breaks the url of any css or js files in the templates folder.

**Comment by rasteiner** [comment]
> Thanks for this however, it breaks the url of any css or js files in the templates folder.

yes, though it looks like `@auto` urls [didn't work with the original version either](https://github.com/schnti/kirby3-cachebuster/issues/8). 

For myself I'm now using a fork of this plugin: https://github.com/rasteiner/kirby3-cachebuster which combines [suggestions from carstenjaksch](https://github.com/schnti/kirby3-cachebuster/pull/12) and adds the `F::exists` check to it. 
Should work with external urls and `@auto`. At least it seems to do so for me :)

The updated file is here: https://github.com/rasteiner/kirby3-cachebuster/blob/master/index.php

**Comment by apps4research** [comment]
Thanks rasteiner, much appreciated :)

**Comment by schnti** [comment]
Fixed in [1.0.4](https://github.com/schnti/kirby3-cachebuster/releases/tag/1.0.4), thanks for the report and sorry it took this long.

Your suggestion — skipping URLs that have a `host` part — was the right instinct. I went a little further, because skipping every absolute URL would have blocked the fix for #8: Kirby passes absolute URLs of your *own* site for automatically loaded template assets, and those should still get a timestamp.

The plugin now only rewrites URLs that resolve to an existing local file. External URLs, protocol-relative URLs and anything carrying a query string or fragment are returned unchanged.
